pages/linux/gsettings.md

```diff
@@ -0,0 +1,32 @@
+# gsettings
+
+> query and modify dconf settings with schema validation
```

Comment:
What in the world is a schema validation, other than https://duckduckgo.com/?t=ffab&q=schema+validation 

Is that the average user will understand, because I don't think so.

Reply:
`gsettings set` will fail if you try to write a key that shouldn't exist, or supply a value that isn't in the range allowed by the schema. `dconf write` doesn't know about schemas and will let you write anything.

Reply:
I extended the `gsettings set` a little to mention this.

Reply:
I don't think its required or could just be shortened to `valid input` 

cc @SethFalco @navarroaxel

Reply:
I think most developers, sysadmins, and power users will probably understand or infer it.
It'd probably be fair to assume others might have to look it up.

I have no opinion on if it should be changed, though. Subjectively, I like it. ^-^'

Both are perfectly valid options, they're just different levels of explicitness:
`valid input` - Input is being validated _somehow_.
`schema validation` - Input is being validated against a schema. (A data model definition, like [JSON Schema](https://json-schema.org/) or [XSD](https://en.wikipedia.org/wiki/XML_Schema_(W3C)).)

Reply:
(https://github.com/tldr-pages/tldr/pull/6251#discussion_r673337616)